libdevcrypto/Common.cpp

```diff
@@ -67,13 +67,20 @@ bool dev::SignatureStruct::isValid() const noexcept
 	return true;
 }
 
+bool dev::SignatureStruct::isZeroSignature() const noexcept
+{
+	return (r == h256(0) && s == h256(0) && v == 0);
```

Comment:
You can save us some time by not putting `0` in the constructor. The default constructor creates "null" hash. You can do one of 4:
```
r == h256()
r == h256{}
r == {}
!r
```
Up to you. C++.